frontend/public/robots.txt

```diff
@@ -3,6 +3,8 @@ User-agent: *
 Disallow: /patterns
 Disallow: /whats-on
 Disallow: /join-challenger
+Disallow: /bundles
+Disallow: /bundles/thankyou
```

Comment:
In principal, I think just the first line (`Disallow: /bundles`) is suffient:

```
The "Disallow" field specifies a partial URI that is not to be visited. This can be a full path,
or a partial path; any URI that starts with this value will not be retrieved. For example,

    Disallow: /help disallows both /help.html and /help/index.html, whereas
    Disallow: /help/ would disallow /help/index.html but allow /help.html. 
```

https://www.w3.org/TR/html4/appendix/notes.html#h-B.4.1.1